libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1230,6 +1248,28 @@ string YulUtilFunctions::resizeDynamicByteArrayFunction(ArrayType const& _type)
 	});
 }
 
+string YulUtilFunctions::cleanUpDynamicByteArrayEndFunction(ArrayType const& _type)
+{
+	solAssert(_type.isByteArray(), "");
+	solAssert(_type.isDynamicallySized(), "");
+
+	string functionName = "celan_up_byte_array_" + _type.identifier();
```

Comment:
```suggestion
	string functionName = "clean_up_byte_array_" + _type.identifier();
```

Reply:
I'm wondering if this might clash with the naming of the generic "type cleanup" function

Reply:
So maybe at least add an "end" to the name.

Reply:
Added "end" to the name, should be better now